package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0-alpha.1",
   "description": "",
   "main": "index.js",
+  "repository": {
```

Comment:
Let's be fancy and just use `"repository": "facebook/fbjs"` since it works :). As you have (and we have in React too I think), it might not be quite right - https://docs.npmjs.com/files/package.json#repository